compiler/cmdlinehelper.nim

```diff
@@ -0,0 +1,85 @@
+## Helpers for binaries that use compiler passes, eg: nim, nimsuggest, nimfix
+
+# TODO: nimfix should use this; currently out of sync
+
+import
+  compiler/[options, idents, nimconf, scriptconfig, extccomp, commands, msgs, lineinfos, modulegraphs, condsyms],
+  std/os
+
+type NimProg* = ref object
+  name*: string
```

Comment:
Remove this field and pass `name` as a parameter to `initDefinesProg` instead.

Reply:
done